doc/requirements.txt

```diff
@@ -5,7 +5,7 @@ matplotlib-inline
 mpmath
 myst-parser
 scipy
-Sphinx
+Sphinx>=3.5.0
```

Comment:
I'm sure we require a much newer version of Sphinx than this.

Reply:
I could see your build uses the quasi latest version as the PDF for 1.13.1 displayed some styling improvements in PDF output for admonitions, which were very recently merged (at Sphinx 7.4.0, which is very recent -- the latest release was done today and is 8.0.0 which has dropped deprecated branches).

Reply:
I added this `>=3.5.0` only to make sure `verbatimforcewraps` existed but if the advice to contributors/testers for builds is always to use latest Sphinx, then this should be dropped of course.